### ISSUE TITLE
[line] builds line markers

### ISSUE THREAD

**Issue by tpwalke2** [tpwalke2/BlueMapSignMarkers#7]

**Comment by elboletaire** [comment]
This is to track a way to add lines via in-game markers too? And same for the area issue I guess (?)

**Comment by tpwalke2** [comment]
@elboletaire, yes, this would be a way to use multiple signs in-game to make a line appear on the web map.

**Comment by elboletaire** [comment]
Nice, I was asking because I came here with plans to add an issue for it :B

BTW thanks for creating this ❤️! it was a must for me switching from overviewer to bluemap, since I have a map with dozens of markers.
